iati/core/tests/test_rulesets.py

```diff
@@ -222,54 +232,66 @@ def rule_constructor(self, rule_type):
             Differs from fixture of same name in TestRuleSubclass as specifies rule_type via specific TestRuleSubclass fixture.
 
         """
-        return iati.core.rulesets.locate_constructor_for_rule_type(rule_type)
+        return iati.core.rulesets.constructor_for_rule_type(rule_type)
+
+    @pytest.fixture
+    def valid_condition_rule(self, rule_constructor, valid_single_context, valid_condition_case):
+        """Return a Rule with a `condition` case."""
```

Comment:
A `Rule` is an instance of a `case`, so `case` can be removed from the docstring.

The same for the `invalid_consition_rule()` docstring.

Reply:
`case` is also a specific structure so it's hard not to use it without creating ambiguity. Have added Todo about this generally and updated docstrings but left the rest as is for now.